GlobalStaticVersion.props

```diff
@@ -8,12 +8,12 @@
     <SemanticVersionMajor>2</SemanticVersionMajor>
     <SemanticVersionMinor>4</SemanticVersionMinor>
     <SemanticVersionPatch>0</SemanticVersionPatch>
-    <PreReleaseMilestone>beta2</PreReleaseMilestone>
+    <PreReleaseMilestone></PreReleaseMilestone>
     <!-- 
       Date when Semantic Version was changed. 
       Update for every public release.
     -->
-    <SemanticVersionDate>2017-05-07</SemanticVersionDate>
+    <SemanticVersionDate>2017-08-04</SemanticVersionDate>
```

Comment:
do not change date when you are not changing major, minor or patch. It will break file versions